torch/quantization/ns/graph_matcher.py

```diff
@@ -87,12 +88,47 @@ def get_non_matchable_modules() -> Set[Callable]:
         torch.quantization.FakeQuantizeBase,
     ])
 
-class _NSGraphMatchableNodesIterator:
+def get_reversed_fusions() -> Set[Tuple[Callable, Callable]]:
```

Comment:
is this only going to support matching 2 nodes?

Reply:
the main restriction is that the subgraph needs one start point and one end point.  Support for a length of 3 nodes can be added here with low eng cost.  This type would become `Union[Tuple[Callable, Callable], Tuple[Callable, Callable, Callable]]`.

note: currently this is not a list because lists are not hashable.  In general, I'm flexible on the exact data structure, happy to revise it if there is a better solution.